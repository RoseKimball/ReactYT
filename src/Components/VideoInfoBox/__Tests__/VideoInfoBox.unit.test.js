import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, mount, render } from 'enzyme';
import VideoInfoBox from '../VideoInfoBox';

describe('VideoInfoBox', () => {
    test('renders VideoInfoBox', () => {
        const wrapper = shallow (
            <VideoInfoBox />
        )
        expect(wrapper).toMatchSnapshot();
    })
    test('renders expanded', () => {
        const wrapper = shallow (
            <VideoInfoBox />
        )
        wrapper.setState({collapsed: false});
        expect(wrapper).toMatchSnapshot();
    })
})