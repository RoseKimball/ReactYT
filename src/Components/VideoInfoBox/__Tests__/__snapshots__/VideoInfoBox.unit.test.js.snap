// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoInfoBox renders VideoInfoBox 1`] = `ShallowWrapper {}`;

exports[`VideoInfoBox renders expanded 1`] = `ShallowWrapper {}`;
